### PR TITLE
Fixed Graph Drawing Bug

### PR DESCRIPTION
**Pull request by ReeceHughes**
Fixed bug where the graph lines were moving around after being drawn.  It was fixed by disabling graph animation.  It worked with animation before so the bug is not 100% resolved, but this will work for now.

### CODE REVIEW

File: server/resources/js/app/device_metric_graphs.js [hunk 1]
```diff
@@ -113,10 +113,10 @@ class MetricGrapher {
     this.memoryUsageHistory_.push(this.metrics.memUsage / 1024.0);
     this.memoryUsageHistory_.shift();
 
-    this.networkSentHistory_.push(this.metrics.netSentPerSec);
+    this.networkSentHistory_.push(this.metrics.netSentPerSec / 1024.0);
```
Comment (cnavrides) [inline server/resources/js/app/device_metric_graphs.js]: why 1024? May be good to have this as a constant and comment about why it's 1024

File: server/resources/js/app/device_metric_graphs.js [hunk 2]
```diff
@@ -162,6 +170,9 @@ class MetricGrapher {
       hover: {
         mode: null
       },
+      animation: {
+            duration: 0,
+        },
```
Comment (sumnerevans) [inline server/resources/js/app/device_metric_graphs.js]: Indentation here is off. Should be 2 spaces.

File: server/template/current.xhtml [hunk 3]
```diff
@@ -23,20 +23,20 @@
...
       <div class="metrics-graph">
-        <h4>CPU Usage</h4>
+        <h4>CPU Usage (%)</h4>
         <canvas id="cpu-usage-graph">
         </canvas>
       </div>
 
       <div class="metrics-graph">
-        <h4>Memory Usage</h4>
+        <h4>Memory Usage (MB)</h4>
```
Comment (sumnerevans) [inline server/template/current.xhtml]: Wrong. Should be MiB (https://en.wikipedia.org/wiki/Mebibyte).

File: server/template/current.xhtml [hunk 4]
```diff
@@ -23,20 +23,20 @@
...
         <canvas id="mem-usage-graph"></canvas>
       </div>
 
       <div class="metrics-graph">
         <div class="row">
           <div class="col-md-8">
-            <h4>Network Usage</h4>
+            <h4>Network Usage (KB/s)</h4>
```
Comment (sumnerevans) [inline server/template/current.xhtml]: Should be KiB/s. We can also consider reporting this in Kbps (kilobits/sec).

**Pull request merged**